src/app/domains/date/date.helpers.tsx

```diff
@@ -0,0 +1,41 @@
+export function parseTime(inputTime: string) {
```

Comment:
suggestion: in my personal projects I usually name this kind of functions (transform machine data into a more readable format by humans) as `friendlyXXXX`. In this case `friendlyTime()` This means that the result is always some kind of customized string.

Reply:
I'm stealing that hehe